src/shared/styles/index.scss

```diff
@@ -1,5 +1,5 @@
 @import './variables.scss';
-@import './theme-gatsby-default.scss';
```

Comment:
@raed-emiketic `theme-gatsby-default` should be placed back in the starter kit unlike customer project where we remove it